next-cloudinary/src/components/CldVideoPlayer/CldVideoPlayer.types.ts

```diff
@@ -2,14 +2,13 @@ import { MutableRefObject } from 'react';
 import {
   CloudinaryVideoPlayer,
   CloudinaryVideoPlayerOptionsLogo,
-  CloudinaryVideoPlayerOptions
 } from '@cloudinary-util/types';
-import { ConfigOptions } from "@cloudinary-util/url-loader";
+import { ConfigOptions, GetVideoPlayerOptions } from "@cloudinary-util/url-loader";
 
 import { GetCldImageUrlOptions } from '../../helpers/getCldImageUrl';
 import { GetCldVideoUrlOptions } from '../../helpers/getCldVideoUrl';
 
-export type CldVideoPlayerProps = Omit<CloudinaryVideoPlayerOptions, "cloud_name" | "autoplayMode" | "publicId" | "secure" | "showLogo" | "logoImageUrl" | "logoOnclickUrl"> & {
+export type CldVideoPlayerProps = Omit<GetVideoPlayerOptions, "cloud_name" | "autoplayMode" | "publicId" | "secure" | "showLogo" | "logoImageUrl" | "logoOnclickUrl"> & {
```

Comment:
`CloudinaryVideoPlayerOptions` is more correct in this example though, `GetVideoPlayerOptions` is an internal API and may not have the same set of parameters that `CloudinaryVideoPlayerOptions` has, because `GetVideoPlayerOptions` takes the user props from `CldVideoPlayerProps` and translates it into `CloudinaryVideoPlayerOptions` which is the source Cloudinary Video Player types

Reply:
Hm, we might want to tweak the props that we Omit here (or switch to Pick or smth) but it seems to me that we quite specifically depend here on `GetVideoPlayerOptions`. Maybe it's a semi-internal thing but we pass our props to a function that expects it here:
https://github.com/cloudinary-community/next-cloudinary/blob/main/next-cloudinary/src/components/CldVideoPlayer/CldVideoPlayer.tsx#L34

There is no translation of props in the middle here.

Reply:
i think i was misreading where we were at here and thinking this was the top level component level change, i think this change makes sense. thanks